ecs-cli/modules/cli/compose/project/project.go

```diff
@@ -134,7 +135,8 @@ func (p *ecsProject) transformTaskDefinition() error {
 	// convert to task definition
 	logrus.Debug("Transforming yaml to task definition...")
 	taskDefinitionName := utils.GetTaskDefinitionName(context.ECSParams.ComposeProjectNamePrefix, context.Context.ProjectName)
-	taskDefinition, err := utils.ConvertToTaskDefinition(taskDefinitionName, &context.Context, p.ServiceConfigs())
+	taskRoleArn := context.CLIContext.String(command.TaskRoleArnFlag)
```

Comment:
This needs to be `context.CLIContext.GlobalString` since the command is going to be `ecs-cli compose --task-role-arn <arn> service up`